src/components/avatars/OcAvatar.vue

```diff
@@ -1,31 +1,50 @@
 <template>
-  <avatar
-    class="oc-avatar"
-    :username="userName"
-    :size="width"
-    :src="src"
+  <span
+    class="vue-avatar--wrapper oc-avatar"
+    :style="[style]"
```

Comment:
Why are the styles wrapped into an array? 🤔

Reply:
Leftovers from the vue-avatar copy-pasta, thanks I'll get rid of it